src/Caching/StackExchangeRedis/src/RedisCache.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace Microsoft.Extensions.Caching.StackExchangeRedis
 {
-    public class RedisCache : IDistributedCache, IDisposable
+    public class RedisCache : IDistributedCache
```

Comment:
@Tratcher @BrennanConroy - Are we concerned about API breaking changes in this library?

Reply:
Yes, we should avoid any API breaks. In this case it's easier to leave an empty Dispose method.